tutorials/Working_with_fNIRS_data.py

```diff
@@ -0,0 +1,124 @@
+"""
+.. _tut_fnirs_data
+
+Use MNE with fNIRS data
+=======================
+# Author: Matteo Caffini  
+# Contact: <matteo.caffini@unitn.it>
+
+"""
+import numpy as np
+
+import mne
+
+raw_ndarray = np.load('/home/matteo/raw_ndarray.npy')
```

Comment:
Yes, @Eric89GXL that would be an option. If @agramfort still prefers a public dataset, I'll stick with him and adapt the tutorial.
BTW, the real file format is a csv file with fluences data, I manually crunched it to get HbO and HbR concentrations and then saved it in numpy format.